designs/panorama_plugin/panorama_plugin.md

```diff
@@ -0,0 +1,60 @@
+# Panorama Security Plugin
+This is a security appliance plugin for the Panorama Security Management Appliance, which orchestrates PaloAlto virtual firewalls deployed by OSC. The plugin shall conform to the [security manager api](https://github.com/opensecuritycontroller/security-mgr-api) specification. We propose to have:
+- A Panorama Dynamic Address Group for each OSC security policy. It will be labeled by a tag and targeted by a _Panorama_ security policy.
```

Comment:
Add note here to say Policies in OSC are synonymous with "Shared" tags in panorama.